modules/virt-troubleshooting-cert-rotation-parameters.adoc

```diff
@@ -0,0 +1,33 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-troubleshooting-cert-rotation-parameters_{context}"]
+
+= Troubleshooting
+
+Deleting one or more `certConfig` values will cause them to revert to the default values, unless the default values conflict with the conditions noted above. If the default values conflict with those conditions, you will receive an error.
+
+If you remove the `server.duration` value in the example below, the default value, `24h0m0s` is greater than the value of `ca.duration`, conflicting with the specified conditions.
```

Comment:
s/example below/following example

s/default value,/default value of

I might move this sentence into a callout where the `<1>` is placed after `server.duration`

Reply:
Adjusting first two, leaving the placement of line 12, the info feels confusing after the example when I attempt it.